app/models/user.rb

```diff
@@ -167,7 +162,7 @@ def hide_checkin_comment?(checkin)
   def get_large_avatar_image(provider, image_url)
     url = case provider
           when 'twitter'  then image_url.sub('_normal', '')
-          when 'facebook' then "#{image_url}?type=large"
+          when 'facebook' then "#{image_url.sub("http://", "https://")}?type=large"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Prefer single-quoted strings inside interpolations.